src/ExceptionHandler.php

```diff
@@ -297,7 +298,18 @@ public function report(Exception $e): void
             }, $e->getErrors());
         }
 
-        $this->logger->error($e->getMessage(), $context);
+        $this->isWarning($e)
+            ? $this->logger->warning($e->getMessage(), $context)
+            : $this->logger->error($e->getMessage(), $context);
+    }
+
+    private function isWarning(Exception $exception): bool
```

Comment:
Lets use the `Throwable` interface here
```suggestion
    private function isWarning(Throwable $exception): bool
```

Reply:
It will never be anything else than an Exception though. This is only called in the report method that we overwrite from Laravel. According to the interface that it implements, the report method will always receive an instance of Exception.